tours/tour.py

```diff
@@ -192,6 +193,50 @@ def get_destination(self, origin):
         destination = locations[m]
         return destination
 
+    def get_secondary_destination(self, origin, destination, empty_location):
```

Comment:
In general, I think this function could benefit of some sort of `maxBy` function, but not sure how this works in Python.

Reply:
I will look into this later, thanks!